packages/tools/devtools/devtools-core/src/ContainerRuntimeDevtools.ts

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IContainerRuntime } from "@fluidframework/container-runtime-definitions/internal";
+import type { IFluidLoadable } from "@fluidframework/core-interfaces";
+
+import { BaseDevtools } from "./BaseDevtools.js";
+import type { ContainerKey } from "./CommonInterfaces.js";
+import type { DecomposedContainer } from "./DecomposedContainer.js";
+import type { ContainerDevtoolsFeatureFlags } from "./Features.js";
+import type { InboundHandlers } from "./messaging/index.js";
+
+/**
+ * Properties for registering a container runtime with the Devtools.
+ * @alpha
+ */
+export interface ContainerRuntimeProps {
+	/**
+	 * The container runtime to register with the Devtools.
+	 */
+	runtime: IContainerRuntime;
```

Comment:
Nit: since this type is used strictly as input to the library, I would recommend tagging the interface as `@input` and making its properties `readonly`.